### PR TITLE
Issue 5640 abjustlayout

### LINKED ISSUE #5640: The text and layout need to be adjusted for better readability and usability across different screen sizes.
### Description
On smaller screens, the content appears cramped and misaligned, making it difficult to read and navigate.

#### Steps to Reproduce
Open the URL https://layer5.io/learn/service-mesh-labs/working-with-meshery-and-istio on a mobile device or resize your browser to a smaller screen width.

### Expected Behavior
- The page should be fully responsive and display well on all devices, including desktops, tablets, and smartphones.
- The text should be readable, and the layout should be user-friendly on smaller screens.

### Screenshots
Card

### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by pukarlamichhane**
**Description**

This PR fixes #5640 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5087bb0acb8f84a7aaa6879551a598d158d8c3f3 at: https://668a92c2757d6ae9a040720b--layer5.netlify.app

**Comment by sudhanshutech** [comment]
@pukarlamichhane make sure you add the issue number in pr message not in title so it will link with appropriate issue.
Also ensure you sign your commits please

File: src/sections/Learn/Lab-single/index.js [hunk 1]
```diff
@@ -6,39 +6,32 @@ import { Link } from "gatsby";
 import PageHeader from "../../../reusecore/PageHeader";
 import Button from "../../../reusecore/Button";
 
-
 const LabSinglePage = ({ frontmatter, body }) => {
-
-  // React.useEffect(() => {
```
Comment (sudhanshutech) [inline src/sections/Learn/Lab-single/index.js:12]: why this?

File: src/sections/Learn/Lab-single/LabSinglePageWrapper.style.js [hunk 2]
```diff
@@ -1,72 +1,124 @@
...
 
 const LabSinglePageWrapper = styled.div`
-    margin-bottom: 4rem;
-	display: block;
+  margin-bottom: 4rem;
+  display: block;
+  padding: 0 1rem;
+  color: white; // Added this line to make the text white
 
-	.backBtn {
```
Comment (sudhanshutech) [inline src/sections/Learn/Lab-single/LabSinglePageWrapper.style.js:7]: avoid doing unnecessary lint changes please

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by vishalvivekm** [comment]
@pukarlamichhane 
Thank you for your contribution.
Let's discuss this on Websites' call. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

File: src/sections/Learn/Lab-single/LabSinglePageWrapper.style.js (hunk 2 above)
Comment (pukarlamichhane) [reply src/sections/Learn/Lab-single/LabSinglePageWrapper.style.js:7]: ok sir

**Comment by Ashparshp** [comment]
@pukarlamichhane Make sure to sign-off commits..

**Comment by sudhanshutech** [comment]
@pukarlamichhane any progress further?

**Comment by pukarlamichhane** [comment]
Already completed

**Comment by Ashparshp** [comment]
@pukarlamichhane Thanks for your contribution, let's discuss this on the website's call. Please add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Comment by l5io** [comment]
🚀 Preview for commit da441c06c9984e638aca20a0937fed14a723c5fa at: https://66a02de54c78b354a02dfe8f--layer5.netlify.app

**Comment by sudhanshutech** [comment]
@pukarlamichhane checking again?

**Comment by Ashparshp** [comment]
@pukarlamichhane, Final check..

**Pull request closed without merging**